defaults/main.yml

```
@@ -29,15 +29,29 @@ sentry_dsn:
 google_analytics_tracking_id:
 extra_environment_variables: {}
 
+# Secure Communications (SSL / TLS)
+letsencrypt: true
+letsencrypt_test: false
+
 # Traefik settings
 traefik_dashboard: false
 traefik_extra_config_path: /etc/hub/traefik
 traefik_docker_extra_config_path: /custom_config
+traefik_enable_tls: true
+traefik_certificate_resolver: lets-encrypt
+traefik_extra_env_vars: {}
+traefik_extra_labels: {}
 
 traefik:
   dashboard: "{{ traefik_dashboard }}"
   extra_config_path: "{{ traefik_extra_config_path }}"
   docker_extra_config_path: "{{ traefik_docker_extra_config_path }}"
+  enable_tls: "{{ traefik_enable_tls }}"
+  enable_lets_encrypt: "{{ letsencrypt }}"
+  enable_lets_encrypt_test: "{{ letsencrypt_test }}"
+  certificate_resolver: "{{ traefik_certificate_resolver }}"
```

Ensures certificate resolver is set to an empty string if tls not enabled

```suggestion
  certificate_resolver: "{{ traefik_enable_tls | ternary(traefik_certificate_resolver, '') }}"
```